Tests/Marketplace/zip_packs.py

```diff
@@ -0,0 +1,178 @@
+        zipped_packs: A list of the downloaded packs paths and their corresponding pack names.
+    """
+    zipped_packs = []
+    with ThreadPoolExecutor(max_workers=MAX_THREADS) as executor:
+        for pack in os.scandir(PACKS_FULL_PATH):  # Get all the pack names
+            if pack.name in IGNORED_FILES + GCPConfig.CORE_PACKS_LIST:
+                continue
+            # Search for the pack in the bucket
+            pack_prefix = os.path.join(GCP_PATH, branch_name, circle_build, pack.name)
+            blobs = list(storage_bucket.list_blobs(prefix=pack_prefix))
+            if blobs:
+                blob = get_latest_pack_zip_from_blob(pack.name, blobs)
+                if not blob:
+                    print_warning(f'Failed to get the zip of the pack {pack.name} from GCP')
+                    continue
+                download_path = os.path.join(destination_path, f"{pack.name}.zip")
+                zipped_packs.append({pack.name: download_path})
+                print(f'Downloading pack from GCP: {pack.name}')
+                executor_submit(executor, download_path, blob)
+            else:
+                print_warning(f'Did not find a pack to download with the prefix: {pack_prefix}')
+
+    return zipped_packs
+
+
+def executor_submit(executor, download_path, blob):
+    executor.submit(blob.download_to_filename, download_path)
+
+
+def cleanup(destination_path):
+    """
+    Cleans up the destination path directory by removing everything except the packs zip.
+    Args:
+        destination_path: The destination path.
+    """
+    files_to_remove = [file_.path for file_ in os.scandir(destination_path) if file_.name != ARTIFACT_NAME]
+    for file_ in files_to_remove:
+        if os.path.isdir(file_):
+            shutil.rmtree(file_)
+        else:
+            os.remove(file_)
+
+
+def get_latest_pack_zip_from_blob(pack, blobs):
+    """
+    Returns the latest zip of a pack from a list of blobs.
+    Args:
+        pack: The pack name
+        blobs: The blob list
+
+    Returns:
+        blob: The zip blob of the pack with the latest version.
+    """
+    blob = None
+    blobs = [b for b in blobs if os.path.splitext(os.path.basename(b.name))[0] == pack and b.name.endswith('.zip')]
+    if blobs:
+        blobs = sorted(blobs, key=lambda b: LooseVersion(os.path.basename(os.path.dirname(b.name))), reverse=True)
+        blob = blobs[0]
+
+    return blob
+
+
+def main():
+    option = option_handler()
+    storage_bucket_name = option.bucket_name
+    zip_path = option.zip_path
+    artifacts_path = option.artifacts_path
+    service_account = option.service_account
+    circle_build = option.circle_build
+    branch_name = option.branch_name
+
+    # google cloud storage client initialized
+    storage_client = init_storage_client(service_account)
+    storage_bucket = storage_client.bucket(storage_bucket_name)
+
+    if not circle_build or not branch_name:
+        # Ignore build properties
+        circle_build = ''
```

Comment:
Is it for local development?

Reply:
nope, if it's the master bucket then it should not be there